zerver/openapi/zulip.yaml

```diff
@@ -5846,6 +5846,68 @@ paths:
                     description: |
                       An example JSON error response when attempting to deactivate the only
                       organization owner in an organization
+  /users/me/status:
+    post:
+      operationId: update-self-status
+      summary: Update your status
+      tags: ["users"]
+      description: |
+        Change your status.
+
+        `POST {{ api_url }}/v1/users/me/status`
+      parameters:
```

Comment:
This should be tweaked to document the emoji parameters.

Reply:
I think you want to inherit `EmojiBase`.  (The CI failure is I think `reaction_type` being missing).